possible to disable auth integration entirely (#1230)
mdn/yari: docs/envvars.md

The suggested value is to set this to set
`REACT_APP_KUMA_HOST=localhost.org:8000` and now sign in links go to that
host name instead. That means you can log in _from_ Yari with a single click.

### `REACT_APP_DISABLE_AUTH`

**Default: `false`**

> This entirely removes the sign-in and then `whoami` XHR fetching.
> Useful when using Yari purely for doing content editing and not interested
> in testing how Yari works for end users.

Reviewer: ```suggestion
This removes sign-in and `whoami` XHR fetching.
Useful when using Yari purely for content editing as authentication is then not required.
```
